.github/workflows/build-gems.yml

```diff
@@ -0,0 +1,85 @@
+name: Build Gems
+on:
+  push:
+    branches:
+      - main
+      - "releases/*"
+
+jobs:
+  build-gems:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rubyPlatform: ["aarch64-linux", "x86_64-linux", "arm64-darwin", "x86_64-darwin", "x64-mingw-ucrt"]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Setup Ruby and Rust
+        uses: oxidize-rb/actions/setup-ruby-and-rust@v1
+        with:
+          ruby-version: "3.3"
+          bundler-cache: true
+          cargo-cache: true
+          cargo-vendor: true
+          working-directory: ./temporalio
+          cache-version: v1-${{ matrix.rubyPlatform }}
+
+      - name: Cross compile gems
```

Comment:
I suppose that it will output files to `temporalio/internal/bridge/3.[123]/temporalio_bridge.(extension)`. Is that exact?

Some comments on what this action will do would be useful to future readers.

Reply:
Yes. Not really wanting to re-document what a separate action does here where we use it, but the `internal/bridge.rb` does have version specific loading